test/WebSites/ApiExplorerWebSite/Controllers/ApiExplorerResponseTypeWithApiConventionController.cs

```diff
@@ -27,6 +27,10 @@ public class ApiExplorerResponseTypeWithApiConventionController : Controller
         [ProducesResponseType(403)]
         public IActionResult PostWithConventions() => null;
 
+        [HttpPost]
+        [Produces("application/json", "text/json")]
+        public IActionResult PostWithProduces(Product p) => null;
```

Comment:
So is the gist of the fix that the convention would apply to this method (as written), but would not apply to:

```
[HttpPost]
        [Produces("application/json", "text/json", Type = typeof(FooBar))]
        public IActionResult PostWithProduces(Product p) => null;
```

If so, then I agree 👍

Reply:
Yup.

Reply:
Beans++